### ISSUE TITLE
Problem when cutting AVI files

### ISSUE THREAD

**Issue by bartekmotyl** [bartekmotyl/simple-video-cutter#62]
It seems that after fixing #60 (in release 0.23) there is a problem with cutting AVI files. 
```
Stream map '0:a' matches no streams. To ignore this, add a trailing '?' to the map.
```

**Comment by bartekmotyl** [comment]
Problem can be replicated e.g. with sample video file `TRA3106.avi` that can be found in http://samples.mplayerhq.hu/avi/

**Comment by bartekmotyl** [comment]
Fixed in development branch, will be included in release 0.24
